lib/sequence_diagram/sequencer.rb

```diff
@@ -0,0 +1,72 @@
+require "ostruct"
+
+module SequenceDiagram
+  class Sequencer
+    def self.sequence
```

Comment:
I'm not opposed to that direction, but why not just use `call` since that is a very `Ruby` message to send.